src/Visualization/Python/Render3D/Bbh.py

```diff
@@ -0,0 +1,291 @@
+
+    render_view.Update()
+    pv.ColorBy(transform_1_display, None)
+
+
+def render_bbh(
+    volume_xmf: str,
+    output: str,
+    aha_xmf: str,
+    ahb_xmf: str,
+    time_step: int = 0,
+    animate: bool = False,
+    camera_angle: str = "Side",
+    color_map: str = "Rainbow Uniform",
+    show_grid: bool = False,
+    show_time: bool = False,
+):
+    """Generate Pictures from XMF files for BBH Visualizations
+
+    Generates pictures from BBH runs using the XMF files generated using
+    generate-xdmf. This script requires that the Lapse and SpatialRicciScalar
+    were output in the volume data.
+
+    Arguments:
+
+      volume_xmf: Path to the volume data xmf file.
+      output: Name of output file generated from paraview. Include extensions
+      such as '.png'
+      aha_xmf: Path to the apparent horizon xmf file for object A.
+      ahb_xmf: Path to the apparent horizon xmf file for object B.
+      camera_angle: Specified camera angle, defaults to Side if empty. Other
+      possible angles Top and Wide
+      color_map: Color map for the lapse, defaults to 'Rainbow Uniform'. Other
+      color maps include 'Inferno (matplotlib)', 'Viridis (matplotlib)', etc.
+      show_grid: Shows the grid lines of the domain.
+      show_time: Shows the simulation time.
+
+    To splice all the pictures into a video, try using FFmpeg"""
+    import paraview.simple as pv
+
+    version = pv.GetParaViewVersion()
+    if version < (5, 11) or version > (5, 11):
+        logger.warning(
+            "WARNING: Your Paraview version is not 5.11, "
+            "the script may not work correctly."
+        )
+
+    # Volume Data Visualization
+    volume_files_xmf = pv.XDMFReader(
+        registrationName=volume_xmf, FileNames=[volume_xmf]
+    )
+
+    # Check for Lapse and SpatialRicciScalar
+    variables = list(volume_files_xmf.PointData.keys())
+    assert (
+        "Lapse" in variables
+    ), "Lapse not found in volume data, the script will not work correctly."
+    assert "SpatialRicciScalar" in variables, (
+        "SpatialRicciScalar not found in volume data, the script will not work"
+        " correctly."
+    )
+
+    render_view = pv.GetActiveViewOrCreate("RenderView")
+
+    # Color the grid
+    color_transfer_function = pv.GetColorTransferFunction("Lapse")
+    color_transfer_function.Discretize = 0
+    color_transfer_function.ApplyPreset(color_map, True)
+    color_transfer_function.InvertTransferFunction()
+
+    # Slice volume data
+    slice = pv.Slice(registrationName="slice", Input=volume_files_xmf)
+    slice.SliceType = "Plane"
+    slice.HyperTreeGridSlicer = "Plane"
+    slice.SliceOffsetValues = [0.0]
+    slice.SliceType.Normal = [0.0, 0.0, 1.0]
+    slice.Triangulatetheslice = 0
+
+    # Warp grid by spatial ricci scalar
+    warp_by_scalar = pv.WarpByScalar(
+        registrationName="WarpByScalar", Input=slice
+    )
+    warp_by_scalar.Scalars = ["POINTS", "SpatialRicciScalar"]
```

Comment:
Mind adding `SpatialRicciScalar` to the output in `Inspiral.yaml`, so we can actually use this visualization script with default output? Or make `Lapse` and `SpatialRicciScalar` optional in this script.